apps/framework-cli/src/infrastructure/olap/clickhouse/queries.rs

```diff
@@ -361,7 +361,9 @@ impl ClickhouseEngine {
             }
             if let Some(d) = is_deleted {
                 // Only add is_deleted if ver is present (validated elsewhere)
-                params.push(format!("'{}'", d));
+                if ver.is_some() {
```

Comment:
if this is validated elsewhere, then when ver is `None`, we should crash and expose the bug